pkgs/development/libraries/libff/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, cmake, boost, gmp, openssl, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  name = "libff-${version}";
+  version = "1.0.0";
+  src = fetchFromGitHub {
+    owner = "scipr-lab";
+    repo = "libff";
+    rev = "v${version}";
+    sha256 = "0dczi829497vqlmn6n4fgi89bc2h9f13gx30av5z2h6ikik7crgn";
+    fetchSubmodules = true;
+  };
+
+  cmakeFlags = [ "-DWITH_PROCPS=Off" ];
+
+  nativeBuildInputs = [ cmake pkgconfig ];
+  buildInputs = [ boost gmp openssl ];
+  enableParallelBuilding = true;
```

Comment:
```suggestion
```

Reply:
Just a question because I'm curious: Why remove this?  It seems like parallel building should be enabled if possible?